create-stack.sh

```diff
@@ -4,33 +4,76 @@ echo "This command will install the analytic stack in namespace ${NS}"
 echo -n "Press enter or ^c to cancel: "
 read
 set -x
-argocd app create clickhouse-operator \
```

Comment:
Rather than override this we should provide installation alternatives. The advantage of using argocd commands is that users can just refer to the cloud and load selected parts of the stack quickly.

Reply:
it still uses argocd commands in a for loop
```
declare -a applications=(prometheus-operator-crds prometheus-rbac prometheus-operator clickhouse-operator prometheus grafana-operator zookeeper clickhouse grafana grafana-datasource)

#helm upgrade -i grafana-operator oci://ghcr.io/grafana-operator/helm-charts/grafana-operator --version v5.3.0 --values apps/grafana-operator/values.yaml --namespace ${NS} --create-namespace
#sleep 5

for apps in "${applications[@]}"; do
  echo "*********** Creating ${apps} ***********"
  argocd app create ${apps} \
    --repo https://github.com/Altinity/argocd-examples-clickhouse.git --path apps/${apps} \
    --dest-server https://kubernetes.default.svc --dest-namespace ${NS} --revision grafana_operator_dashboard
done
```